packages/core/src/build/configAndIndexingFunctions.ts

```diff
@@ -504,20 +505,13 @@ export async function buildConfigAndIndexingFunctions({
         networkName: source.network,
       } as const;
 
-      const resolvedFactory = source?.factory;
       const resolvedAddress = source?.address;
 
-      if (resolvedFactory !== undefined && resolvedAddress !== undefined) {
-        throw new Error(
-          `Validation failed: Contract '${contractMetadata.name}' cannot specify both 'factory' and 'address' options.`,
-        );
-      }
-
-      if (resolvedFactory) {
+      if (typeof resolvedAddress === "object") {
```

Comment:
Are we validating the factory arguments at runtime anywhere?

Reply:
https://github.com/ponder-sh/ponder/blob/kjs/factory/packages/core/src/build/factory.ts